### ISSUE TITLE
No exports condition for Svelte (Deprecation Warning)

### ISSUE THREAD

**Issue by bshore** [iconify/iconify#261]
Howdy!

I've just upgraded one of my SvelteKit projects to the newest major (`@sveltejs/kit@2.0.0`) and I'm seeing a console warning when starting up my project.

```
WARNING: The following packages have a svelte field in their package.json but no exports condition for svelte.

@iconify/svelte@3.1.4

Please see https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/faq.md#missing-exports-condition for details.
```
Following the link displays this information:

> If you see a warning logged for this when using a Svelte library, please tell the library maintainers.
Using the svelte field in package.json to point at .svelte source files is deprecated and you must use a svelte [export condition](https://nodejs.org/api/packages.html#conditional-exports). vite-plugin-svelte 3 still resolves it as a fallback, but in a future major release this is going to be removed and without exports condition resolving the library is going to fail.
```diff
// package.json
  "files": ["dist"],
  "svelte": "dist/index.js",
+ "exports": {
+   ".": {
+     "svelte": "./dist/index.js"
+   }
  }
```

**Comment by Rar9** [comment]
+1 please update

**Comment by cyberalien** [comment]
Added in 3.1.5

**Comment by cyberalien** [comment]
3.1.5 fix was broken, fixed correctly in 3.1.6
